pytest-selenium/pages/content.py

```diff
@@ -27,6 +31,55 @@ def toolbar(self):
     def sidebar(self):
         return self.SideBar(self)
 
+    @property
+    def attribution(self):
+        return self.Attribution(self)
+
+    @property
+    def next_link(self):
+        return self.find_element(*self._next_locator)
+
+    def next_click(self):
+            self.offscreen_click(self.next_link)
+            
+
+    @property
+    def back_link(self):
+        return self.find_element(*self._back_locator)
+
+    def back_click(self):
+            self.offscreen_click(self.back_link)
+            
+
+    @property
+    def current_url(self):
+        return self.driver.current_url
+     
+
+
+    class Attribution(Region):
+        _root_locator = (By.XPATH,"//summary[contains(@class,'Attribution__Summary')]")
```

Comment:
can we use css selector instead of xpath? i think they're in general more readable

Reply:
also i'm pretty sure there is a testid on this now

Reply:
Yes, I added one.